chains/container-chains/runtime-templates/frontier/src/xcm_config.rs

```diff
@@ -256,6 +258,7 @@ impl pallet_xcm::Config for Runtime {
     // TODO pallet-xcm weights
     type WeightInfo = weights::pallet_xcm::SubstrateWeight<Runtime>;
     type AdminOrigin = EnsureRoot<AccountId>;
+    type AuthorizedAliasConsideration = (); // TODO: revisit this
```

Comment:
This one I don't understand. However () is not the same as Disabled, () always returns Ok while Disabled always returns Err

https://github.com/paritytech/polkadot-sdk/pull/6336

https://github.com/paritytech/polkadot-sdk/blob/a51c31246a789078f6109428974a31e2af55e6b2/polkadot/runtime/westend/src/xcm_config.rs#L325

https://github.com/paritytech/polkadot-sdk/blob/a51c31246a789078f6109428974a31e2af55e6b2/cumulus/parachains/runtimes/assets/asset-hub-westend/src/xcm_config.rs#L582